Tests/SwiftRefactorTest/MigrateToNewIfLetSyntax.swift

```diff
@@ -17,102 +17,113 @@ import SwiftSyntaxBuilder
 import XCTest
 import _SwiftSyntaxTestSupport
 
+@discardableResult
+func testRefactorIfLet(
+  _ syntax: ExprSyntax,
+  expected: ExprSyntax,
+  file: StaticString = #file,
+  line: UInt = #line
+) throws -> ExprSyntax {
```

Comment:
I think it would be cleaner to name this `AssertRefactorIfLet` and not have it return a value. Since we are asserting that `refactored` is `expected` callers don’t actually need the return value and can use whatever they are passing to `expected` instead.